chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/SEDRow.java

```diff
@@ -0,0 +1,41 @@
+package gov.healthit.chpl.domain;
```

Comment:
The domain package is for classes that get sent back and forth between the user/UI and the API. I don't think SEDRow counts. Could it be moved to one of the app packages since it's specific to the SED app?